### PR TITLE
feat(base): create movie model with serializer

### PR DESCRIPTION
**Pull request by robintsunny**
what: db movie model
why: model for movies

### CODE REVIEW

**Review by jesscxu — APPROVED** [review]
LGTM can you check again that `.DS_Store` is in the `.gitignore` file? its still showing up

File: lib/models/movie.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Bookshelf = require('../libraries/bookshelf');
+
+module.exports = Bookshelf.Model.extend({
+  tableName: 'movies',
+  serialize: function () {
```
Comment (ajpawlicki) [inline lib/models/movie.js:7]: Why is this called `serialize`?

File: lib/models/movie.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    return {
+      id: this.get('id'),
+      title: this.get('title'),
+      release_year: this.get('release_year'),
```
Comment (ajpawlicki) [inline lib/models/movie.js:11]: Are these db lookups?

**Review by ajpawlicki — APPROVED** [review]
LGTM asked a couple questions because I never actually did this but am curious.

File: lib/models/movie.js (hunk 2 above)
Comment (robintsunny) [reply lib/models/movie.js:11]: i think serialize takes in an instance of a movie and then sets a db entry with the corresponding values

File: lib/models/movie.js (hunk 1 above)
Comment (robintsunny) [reply lib/models/movie.js:7]: thats what the tutorial told me to call it. but i think it's to create a movie object and then save it to the db

Comment (jesscxu) [reply lib/models/movie.js:7]: that is the convention so for a bookshelf model, the serialize function gets called when the controller is done running the output matches what is defined by serialize

File: lib/models/movie.js (hunk 2 above)
Comment (jesscxu) [reply lib/models/movie.js:11]: this is just retrieving the values of the model ... not accessing the db

Comment (ajpawlicki) [reply lib/models/movie.js:11]: Thx

File: lib/models/movie.js (hunk 1 above)
Comment (ajpawlicki) [reply lib/models/movie.js:7]: So the output of the controller has to match the output of `serialize`?

**Pull request merged**